Sources/MapboxNavigation/InstructionPresenter.swift

```diff
@@ -179,7 +206,8 @@ class InstructionPresenter {
             attachment.image = image
         } else {
             let view = GenericRouteShield(pointSize: dataSource.font.pointSize, text: text)
```

Comment:
Should the font also come from the appearance proxy?

Reply:
Font is applied to `NSTextAttachment` afterwards, so I don't think it's relevant for cache. I'll run some manual tests though and consider adding unit-test.